### ISSUE TITLE
Coordinate merging and testing of veteran_status eMIS removal

### ISSUE THREAD

**Issue by rmtolmach** [department-of-veterans-affairs/va.gov-team#76033]
### Description and background
The bulk of the work for https://github.com/department-of-veterans-affairs/va.gov-team/issues/75657 is done. The PRs are up and have been partially reviewed. Managing the merging of the vest-website PR and the vets-api PR has taken longer than expected. This issue covers that work

### Tasks
- [x] Get required approvals for https://github.com/department-of-veterans-affairs/vets-website/pull/27851
    - [x] tests passing
- [x] Get required approvals for https://github.com/department-of-veterans-affairs/vets-api/pull/15393
    - [x] tests passing
- [x] Merge vets-website PR
- [x] Merge vets-api PR  after vets-website PR so vets-api deploys after vets-website
- [x] Clean up TODOs [from index.js](https://github.com/department-of-veterans-affairs/vets-website/pull/27851/files#diff-69b42361c09c4f5fe686c0a974d713a6efdd76de85df524c87d3f9b12e9f1e00)
    - [x] test locally. 

### Acceptance criteria
- [x] ArgoCD environments sync without errors after code is merged.
- [x] Watch [this alert](https://vagov.ddog-gov.com/synthetics/details/aji-ays-bjp?monitorStatus=ok&from_ts=1707396804105&to_ts=1707412012421&live=true) and make sure it does not fire after code is merged in staging.  
- [x] Cleanup vets-website PR is merged.

**Comment by jennb33** [comment]
@rmtolmach is waiting for CI to "get better" - the tests are taking 2 hours to run. She will figure out to do more work on her branch in the interim.

**Comment by rmtolmach** [comment]
For vets-website PR, https://github.com/department-of-veterans-affairs/vets-website/pull/27851, main branch was merged into my PR yesterday afternoon. Tests are passing. Waiting on all required reviews.

**Comment by rmtolmach** [comment]
PR was merged. Tested in staging. I was able to successfully log in without errors
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/1c118086-3c26-4755-ac58-830cb7708a76)

The error object in the user response still says EMIS which is expected because the backend is still sending "EMIS". After the vets-api PR merges, it should change to "VAProfile".

Next up: get vets-api PR approved and merged.

**Comment by rmtolmach** [comment]
**Current Status**: vets-website PR merged and deployed to prod 2/15. vets-api PR merged and deployed to dev and staging (around noon ET).
User 32. Staging. VA Profile error handled. Sign-in was successful. 
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/a8f2c295-d0b8-455f-a1c5-b09ec07ba37f)

**Comment by rmtolmach** [comment]
- Final cleanup PR https://github.com/department-of-veterans-affairs/vets-website/pull/28049 needs frontend approval. Once it's merged, this ticket can be closed.
